docs/index.md

```diff
@@ -58,27 +58,29 @@ Get started
 </h1>
 
 <h2>
-Define a component in JavaScript:
+Define a component
 </h2>
 
-<h3 style="margin: 12px 0 0 0;">
-custom-greeting.js
-</h3>
+It's easy to define a Web Component with LitElement:
 
 ```js
-{% include projects/index/custom-greeting.js %}
+{% include projects/index-typescript/custom-greeting.ts %}
```

Comment:
How about a note like `* This example uses TypeScript and decorators. You can also easily write LitElements in plain JavaScript`